Generator/OrmGenerator.php

```diff
@@ -56,7 +56,14 @@ public function generateMappingEntityFiles(OutputInterface $output, BundleMetada
                 $output->writeln(sprintf('   ~ <info>%s</info>', $fileName));
             } else {
                 $output->writeln(sprintf('   + <info>%s</info>', $fileName));
-                copy($src_file, $dest_file);
+
+                $mappingEntityTemplate = file_get_contents($src_file);
+
+                $string = Mustache::replace($mappingEntityTemplate, array(
+                    'namespace' => $bundleMetadata->getExtendedNamespace(),
+                ));
```

Comment:
> We can't use DI without a BC break.

Yes we can : 

- Add an optional argument to the constructor.
- If nothing is provided, complain about the deprecation, and initialize it

Or am I missing something? The LSP does not apply to constructors, ok?